foundation/foundation-mda/src/main/resources/templates/deployment/spark-operator/spark-application-test-values.yaml.vm

```diff
@@ -16,6 +16,24 @@ sparkApp:
         - io.delta:delta-storage:${versionDelta}
         #end
       #end
+    #if (${enableFileStore})
+    #foreach (${fileStore} in ${fileStores})
+    driver:
+      env:
+        - name: "${fileStore.getName()}_FS_PROVIDER"
+          value: "REPLACE ME: YOUR FILE STORE PROVIDER GOES HERE!"
+        - name: "${fileStore.getName()}_FS_ACCESS_KEY_ID"
+          valueFrom:
+            secretKeyRef:
+              name: remote-auth-config
+              key: AWS_ACCESS_KEY_ID
+        - name: "${fileStore.getName()}_FS_SECRET_ACCESS_KEY"
+          valueFrom:
+            secretKeyRef:
+              name: remote-auth-config
+              key: AWS_SECRET_ACCESS_KEY
+    #end
+    #end
```

Comment:
I: @ewilkins-csi @meliz19  Sorry here it is. I've also updated the test script to confirm this behavior as well.